src/app/components/snippet-box/snippet-box.component.html

```diff
@@ -1,46 +1,136 @@
 <div class="title-row">
-    <input type="text" aria-label="Title" class="title" [(ngModel)]="snippet.title" title="Snippet Title" name="title"
-    placeholder="Title" i18n-placeholder="@@title" spellcheck="false" i18n-title="The title for this snippet@@title"
-    i18n-name="@@title" i18n-aria-label="@@title">
-    <input type="text" aria-label="Index" class="index" [(ngModel)]="snippet.index" title="Snippet Index" name="index"
-    placeholder="1" i18n-placeholder="@@index" spellcheck="false" i18n-title="The index for this snippet@@index"
-    i18n-name="@@index" i18n-aria-label="@@index">
+  <input
+    type="text"
+    aria-label="Title"
+    class="title"
+    [(ngModel)]="snippet.title"
+    title="Snippet Title"
+    name="title"
+    placeholder="Title"
+    i18n-placeholder="@@title"
+    spellcheck="false"
+    i18n-title="The title for this snippet@@title"
+    i18n-name="@@title"
+    i18n-aria-label="@@title"
+  />
+  <input
```

Comment:
Would you mind removing this input or commenting it out? I've been meaning to do that, but I haven't gotten around to it.